fe/fe-core/src/main/java/com/starrocks/sql/optimizer/Group.java

```diff
@@ -261,7 +261,12 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return "->  " + "Group: " + id;
+        StringBuilder sb = new StringBuilder();
+        sb.append("->  ").append("Group: ").append(id).append('\n');
+        for (GroupExpression expr : logicalExpressions) {
```

Comment:
I think we only look logical expression when we debug, and maybe will only print `...` because message  is too long if 
 we print physical expression